### ISSUE TITLE
ParsingException: Unknown vox version: 200 expected 150

### ISSUE THREAD

**Issue by AnnanFay** [gromgull/py-vox-io#10]
I guess this library no longer works?

Trying to load a file I just created with latest version gives this error message:

> ParsingException: Unknown vox version: 200 expected 150

**Comment by gromgull** [comment]
yeah, I've not really been maintaining this.... also the file format spec is outdated: https://github.com/ephtracy/voxel-model/blob/master/MagicaVoxel-file-format-vox.txt

I've asked if we can have an update: https://twitter.com/gromgull/status/1767634680323428679

**Comment by gromgull** [comment]
Ah seems @midstreeeam has a more updated version here: https://github.com/midstreeeam/MidVoxIO

**Comment by gromgull** [comment]
https://github.com/midstreeeam/MidVoxIO/issues/3

**Comment by AnnanFay** [comment]
Thanks for the info!

I've tried the MidVoxIO library and it appears to work for version 200 with that small parsing change. Well enough for my purposes anyway.
